recipes/ckan/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "ckan" %}
+{% set version = "2.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ckan/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 1fdc09f4b1ae3a70d84138118c86eee056695235e5be8c42501c8af6b6679686
+  patches:
+    - webob.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python <3.0
+    - setuptools
+  run:
+    - python <3.0
+    - alembic ==1.0.0
+    - babel ==2.3.4
+    - bleach ==3.0.2
+    - click ==6.7
+    - fanstatic >=0.12
+    - flask ==1.1.1
```

Comment:
Do you need such strict requirements? This will make this package uninstallable in a environment with another package that needs `flask >=1.1.2`

Reply:
@isuruf updated all deps to >=

Reply:
@isuruf
I don't fully agree with this change because the strict pinnings were provided by the upstream authors. Also, I think CKAN is something that would mostly be installed in an environment by itself, so it is not necessary for it to be installable with other packages. 

@rluria14,
We definitely should not change *all* the requirements to be >=.

Reply:
@sodre I have reverted back to original pinnings. some of the deps needed to remain as >= due to version availability on the forge.